transport/http/codec.go

```diff
@@ -25,7 +25,7 @@ type EncodeErrorFunc func(http.ResponseWriter, *http.Request, error)
 func DefaultRequestDecoder(r *http.Request, v interface{}) error {
 	codec, ok := CodecForRequest(r, "Content-Type")
 	if !ok {
-		return errors.BadRequest("CODEC", r.Header.Get("Content-Type"))
+		return errors.BadRequest("CODEC unregister Content-Type:", r.Header.Get("Content-Type"))
```

Comment:
should be set in message

Reply:
> should be set in message

There seems to be no other way for me to judge the existence of this problem from the outside of the function:)
it returns the header processing by default

Reply:
```
errors.BadRequest("CODEC", fmt.Sprintf("unregister Content-Type: %s", r.Header.Get("Content-Type")))
```
You can set it in message to provide details of the error.

Reply:
nice